docs/logging.md

```diff
@@ -32,10 +32,12 @@ When the last `CHECK` fails in the "Bar" test case, then only one message will b
 
 `UNSCOPED_INFO` is similar to `INFO` with two key differences:
 
-- Lifetime of unscoped info is not tied to its own scope.
-- Unscoped info can be reported by the first following assertion only, regardless of the result of that assertion.
+- Lifetime of an unscoped message is not tied to its own scope.
+- An unscoped message can be reported by the first following assertion only, regardless of the result of that assertion.
 
-This macro can be used for reporting information from helper functions or inner scopes because lifetime of `UNSCOPED_INFO` is limited by the assertion following whereas lifetime of `INFO` is limited by its own scope. An example:
+In other words, lifetime of `UNSCOPED_INFO` is limited by the following assertion (or by the end of test case/section, whichever comes first) whereas lifetime of `INFO` is limited by its own scope.
+
+These make this macro useful for reporting information from helper functions or inner scopes. An example:
```

Comment:
`These make this macro` is about differences to `INFO`? Probably it might be written like `These differences make this macro` or `That makes this macro`

Reply:
Yes, it refers to properties explained just above. It sounds okay to me, but feel free to rephrase it and push.